### PR TITLE
Create authorization doc

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- Document which actions on Github authorize actions in Prow 
- ...
- ...

### CODE REVIEW

File: docs/prow/authorization.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Members of the `kyma-project` organization are considered trusted users. Trigger starts jobs automatically when a trusted user opens a pull request or commits changes to a pull request branch. Alternatively, trusted collaborators can start jobs manually via the `/test all`, `/test {JOB_NAME}` and `/retest` commands, even if a particular pull request was created by an external user. 
+
+### External contributors
+External contributors are users outside the `kyma-project` organization. Trigger does not automatically start test jobs on pull requests created by external contributors. Furthermore, external contributors are not allowed to manually run tests on their own pull requests.
+
+> **NOTE:** External contributors can still trigger tests on pull requests created by trusted users.
+
+## Authorization decisions enforced by Prow
+
+Action on Prow can be only triggered by webhooks. To configure them you need to provide two secrets:
```
Comment (tomekpapiernik) [inline docs/prow/authorization.md]:
```suggestion
Actions on Prow can be triggered only by webhooks. To configure them you must provide two secrets:
```

Comment (tomekpapiernik) [reply docs/prow/authorization.md]: Where do you "provide the secrets"? SECRETS as in "Kubernetes <Secret> object"?

File: docs/prow/authorization.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- hmac-token - used to validate webhook
```
Comment (tomekpapiernik) [inline docs/prow/authorization.md]:
```suggestion
- `hmac-token` - used to validate webhook
```

File: docs/prow/authorization.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- oauth-token - GitHub bot access token
```
Comment (tomekpapiernik) [inline docs/prow/authorization.md]:
```suggestion
- `oauth-token` - stores the access token used by the GitHub bot
```

File: docs/prow/authorization.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+| Service account name          | Usage                                                      | Required roles |
+| :---------------------------- | :----------------------------------------------------------| :------------- |
+| **sa-gcs-plank**              | Used by Prow plan microservice | `Storage Object Admin`
+| **sa-gke-kyma-integration**   | Running integration tests on GKE cluster | `Compute Admin`, `Kubernetes Engine Admin`, `Kubernetes Engine Cluster Admin`, `DNS Administrator`, `Service Account User`, `Storage Admin`
+| **sa-kyma-artifacts**         | Saving release artifacts to the GCS bucket | `Storage Object Admin`
+| **sa-vm-kyma-integration**    | Running integration tests on minikube | `Compute Instance Admin (beta)`, `Compute OS Admin Login`, `Service Account User`
+| **sa-gcr-push-kyma-project**  | Publishing docker images | `Storage Admin`
+
+## Kubernetes RBAC rules on Prow cluster
```
[the remark this answers is not in the document]
Comment (jakkab) [reply docs/prow/authorization.md]: Done. Section fixed.

File: docs/prow/authorization.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+> **NOTE:** External contributors can still trigger tests on pull requests created by trusted users.
+
+## Authorization decisions enforced by Prow
+
+Actions on Prow can be triggered only by webhooks. To configure them you must create two Github Secrets on your Prow cluster:
+- `hmac-token` - used to validate webhook
+- `oauth-token` - stores the access token used by the GitHub bot
+
+Follow the official [Prow documentation](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#create-the-github-secrets) to learn how create the secrets.
```
Comment (tomekpapiernik) [inline docs/prow/authorization.md]:
```suggestion
Follow the official [Prow documentation](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#create-the-github-secrets) to learn how to create the Secrets.
```

**Pull request merged**